model/ftn/wminitmd.ftn

```diff
@@ -1460,7 +1463,7 @@
         CALL NEXTLN ( COMSTR , MDSI , MDSE2 )
         IF(J .LE. 2) THEN
           OUTFF(J,I)=0
-          WORDS(1:6) =''
+          WORDS(6) =''
```

Comment:
@ukmo-jianguo-li - did you mean to change this line?
This looks like an older version of the code that was changed in PR #166

Reply:
Do not remember I made that change.   Correct it please if you think it is wrong.

Reply:
Done.